Dynatrace/dynatrace-operator
# Add merged entries in helm-index

# Description

Currently we have 2 repos with different helm indices. This PR merges the index of the current repo with the index of the old one.

## How can this be tested?

Add the current helm-repo to your helm installation and install the oneagent-operator. => This should work now.

## Checklist
